parts/k8s/cloud-init/artifacts/sshd_config

```diff
@@ -4,16 +4,22 @@ Port 22
 #ListenAddress ::
 #ListenAddress 0.0.0.0
 Protocol 2
+
+# 5.2.11 Ensure only approved MAC algorithms are used
+MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
```

Comment:
Looks like we aren't validating these 3 configs in the E2E test

Reply:
added